formula/composer.rb

```diff
@@ -0,0 +1,33 @@
+class Composer < Formula
+  desc "Dependency Manager for PHP"
+  homepage "https://getcomposer.org"
+  url "https://getcomposer.org/download/1.6.3/composer.phar"
+  sha256 "52cb7bbbaee720471e3b34c8ae6db53a38f0b759c06078a80080db739e4dcab6"
+  revision 1
```

Comment:
Do we need this?

Reply:
Not necessarily no but @ilovezfs added it to the php formula when it was migrated to force an upgrade so I assumed there was a good reason for that. If there isn't for this Formula we can remove it.